src/components/LayoutFooter/index.tsx

```diff
@@ -1,127 +1,217 @@
+import { ReactComponent as IterativeSVG } from '../../../static/img/iterative-icon.svg'
+
 import styles from './styles.module.css'
 
 const docsPage = getFirstPage()
 
-const LayoutFooter: React.FC<Required<ILayoutModifiable>> = ({ modifiers }) => (
+interface IFooterLinkData {
+  href: string
+  text: string
+  icon?: JSX.Element
+  target?: '_blank'
+}
+
+interface IFooterListData {
+  header: string
+  links: Array<IFooterLinkData>
+}
+
+const footerListsData: Array<IFooterListData> = [
+  {
+    header: 'Product',
+    links: [
+      {
+        href: '/',
+        text: 'Overview'
+      },
+      {
+        href: '/features',
+        text: 'Features'
+      }
+    ]
+  },
+  {
+    header: 'Help',
+    links: [
+      { href: '/support', text: 'Support' },
+      { href: '/doc/start', text: 'Get started' },
+      { href: '/community', text: 'Community' },
+      { href: docsPage, text: 'Documentation' }
+    ]
+  },
+  {
+    header: 'Community',
+    links: [
+      {
+        href: 'https://twitter.com/DVCorg',
+        text: 'Twitter',
+        icon: <TwitterSVG className={styles.icon} />,
+        target: '_blank'
+      },
+      {
+        href: 'https://github.com/iterative/dvc',
+        text: 'Github',
+        icon: <GithubSVG className={styles.icon} />,
+        target: '_blank'
+      },
+      {
+        href: '/chat',
+        text: 'Discord',
+        icon: <DiscordSVG className={styles.icon} />
+      }
+    ]
+  },
+  {
+    header: 'Company',
+    links: [
+      {
+        href: '/blog',
+        text: 'Blog'
+      },
+      {
+        href: '/doc/user-guide/privacy',
+        text: 'Privacy Policy'
+      },
+      {
+        href: 'https://iterative.ai/about#career',
+        text: 'Career',
+        target: '_blank'
+      }
+    ]
+  },
+  {
+    header: 'Other Tools',
+    links: [
+      {
+        href: '/',
+        text: 'DVC',
+        icon: <LogoSVG className={styles.productIcon} />
+      },
+      {
+        href: 'https://cml.dev/',
+        text: 'CML',
+        icon: <CmlSVG className={styles.productIcon} />,
+        target: '_blank'
+      },
+      {
+        href: 'https://studio.iterative.ai/',
+        text: 'Studio',
+        icon: <StudioSVG className={styles.productIcon} />,
+        target: '_blank'
+      }
+    ]
+  }
+]
+
+const footerSocialIconsData: Array<ISocialIcon> = [
+  {
+    site: 'github',
+    label: 'DVC Github Page',
+    url: 'https://github.com/iterative/dvc'
+  },
+  {
+    site: 'twitter',
+    label: 'DVC Twitter',
+    url: 'https://twitter.com/DVCorg'
+  },
+  {
+    site: 'youtube',
+    label: 'DVC.org Youtube Channel',
+    url: 'https://www.youtube.com/channel/UC37rp97Go-xIX3aNFVHhXfQ'
+  },
+  {
+    site: 'linkedinNoBg',
+    label: 'Iterative LinkedIn',
+    url: 'https://www.linkedin.com/company/iterative-ai'
+  },
+  {
+    site: 'discord',
+    label: 'DVC Discord chat',
+    url: 'https://www.dvc.org/chat'
+  }
+]
+
+const FooterLists: React.FC = () => (
```

Comment:
Should I move these functions(`FooterLists` and `FooterSocialIcons`) to a folder inside `LayoutFooter`?

Reply:
Sounds like a sensible refactor, but not necessary before merge at least.